### PR TITLE
Fixed UI bugs

### LINKED ISSUE #2641: Design issues on session, schedule, CfS page
* [ ] On the event page the left sidebar stays there when the user scrolls down. Please implement this on the other pages as well, e.g. https://eventyay.com/e/18252ab6/schedule/
   * [ ] sessions page
   * [ ] schedule page
   * [ ] call for speakers page 
* [ ] The tracks view on schedule page does not show colors of session (https://eventyay.com/e/18252ab6/schedule/)
* [ ] The call for speaker page needs to include the year in the date format: Compare https://eventyay.com/e/18252ab6/cfs/

### PR DESCRIPTION
**Pull request by Marauderer97**
Fixed 3 UI bugs. This will resolve #2641

### CODE REVIEW

**Comment by mariobehling** [comment]
@Marauderer97 Thank you.

Could you provide screenshots and a link to a sample system setup e.g. on Heroku to facilitate the review process, please?

Cheers.

File: app/static/js/admin/event/scheduler.js [hunk 1]
```diff
@@ -253,6 +253,8 @@ function addSessionToTimeline(sessionRef, position, shouldBroadcast) {
     var $mobileSessionElement = $(mobileSessionTemplate);
     $mobileSessionElement.find('.time').text(sessionRefObject.session.start_time.format('hh:mm A'));
     $mobileSessionElement.find('.event').text(sessionRefObject.session.title);
+    updateColor($mobileSessionElement.find('.event'), sessionRefObject.session.track);
+
```
Comment (SaptakS) [inline app/static/js/admin/event/scheduler.js]: Please remove the extra lines.

File: app/templates/gentelella/guest/event/sessions.html [hunk 2]
```diff
@@ -20,6 +20,8 @@ <h1 style="font-weight: 300; font-size: 30px">{{ _("Sessions") }}</h1>
 
 {% block tail_js %}
     {{ super() }}
+    <script src="{{ url_for('static', filename='admin/lib/sticky-kit/jquery.sticky-kit.min.js') }}"></script>
+
```
Comment (SaptakS) [inline app/templates/gentelella/guest/event/sessions.html]: Remove extra line

File: app/templates/gentelella/guest/event/schedule.html [hunk 3]
```diff
@@ -8,6 +8,7 @@
     {{ super() }}
     <link href="{{ url_for('static', filename='css/admin/scheduler.css') }}" rel="stylesheet">
     <link rel="stylesheet" href="{{ url_for('static', filename='css/guest/sessions.css') }}"/>
+
```
Comment (SaptakS) [inline app/templates/gentelella/guest/event/schedule.html]: Remove extra line.

**Comment by Marauderer97** [comment]
@mariobehling @SaptakS 
1. The navbar in cfs page, where it sticks to the page:
image
2. Fixing colors in the tracks view:
image
3. The date in the cfs page(has year now):
image

**Comment by Marauderer97** [comment]
@SaptakS Removed the extra lines :+1:

**Pull request merged**